src/Endpoints/CustomHostnames.php

```diff
@@ -120,16 +120,33 @@ public function getHostname(string $zoneID, string $hostnameID)
 
     /**
      * @SuppressWarnings(PHPMD.BooleanArgumentFlag)
+     * @SuppressWarnings(PHPMD.CyclomaticComplexity)
+     * @SuppressWarnings(PHPMD.NPathComplexity)
```

Comment:
@jacobbednarz I'm unsure how is the best way to refactor this in the way the SDK has been built. Would a protected method to build the query be acceptable, or should I let the PHPMD suppression stay here or any other suggestions you may have?

Reply:
I'm fine with the suppression here as it's pretty clear what the intention is.